backend/src/main/scala/cromwell/backend/standard/callcaching/RootWorkflowFileHashCacheActor.scala

```diff
@@ -95,7 +97,8 @@ class RootWorkflowFileHashCacheActor private(override val ioActor: ActorRef, wor
           case FileHashValueNotRequested =>
             log.error(s"Programmer error! Not expecting a hash request timeout when a hash value has not been requested: ${fileHashContext.file}")
           case v =>
-            log.error(s"Programmer error! Not expecting a hash request timeout when hash value '$v' is already in the cache: ${fileHashContext.file}")
+            log.info(s"Received hash request timeout when hash value '$v' was already in the cache: ${fileHashContext.file}." +
+              "This is normal and happens due to race condition between scheduler thread responsible for firing timeouts and actor's message processing thread.")
```

Comment:
(assuming my understanding is correct)
```suggestion
              "This may occur due to a benign race condition between hash value receipt and timeout.")
```